File: FaceSample/app/src/main/java/com/regula/facesample/items/features/LivenessDisableStepItem.java
package com.regula.facesample.items.features;

import android.content.Context;

import com.regula.facesample.items.CategoryItem;
import com.regula.facesdk.FaceSDK;
import com.regula.facesdk.configuration.LivenessConfiguration;
import com.regula.facesdk.enums.LivenessSkipStep;

public class LivenessDisableStepItem extends CategoryItem {

   @Override
   public void onItemSelected(Context context) {
      LivenessConfiguration configuration = new LivenessConfiguration.Builder()
              .setSkipStep(LivenessSkipStep.START_STEP, LivenessSkipStep.DONE_STEP)
              .build();
      FaceSDK.Instance().startLiveness(context, configuration, livenessResponse -> {});
   }

   @Override
   public String getDescription() {
      return "Hide some steps in liveness process";
   }

   @Override
   public String getTitle() {
      return "Disable steps in liveness";
   }
}

File: FaceSampleKotlin/app/src/main/java/com/regula/facesamplekotlin/items/features/LivenessDisableStepItem.kt
package com.regula.facesamplekotlin.items.features

import android.content.Context
import com.regula.facesamplekotlin.category.CategoryItem
import com.regula.facesdk.FaceSDK
import com.regula.facesdk.configuration.LivenessConfiguration
import com.regula.facesdk.enums.LivenessSkipStep
import com.regula.facesdk.model.results.LivenessResponse

class LivenessDisableStepItem : CategoryItem() {

    override fun onItemSelected(context: Context) {
        val configuration = LivenessConfiguration.Builder()
            .setSkipStep(LivenessSkipStep.START_STEP, LivenessSkipStep.DONE_STEP)
            .build()
        FaceSDK.Instance().startLiveness(
            context, configuration
        ) { livenessResponse: LivenessResponse? -> }
    }

    override val title: String
        get() = "Disable steps in liveness"

    override val description: String
        get() = "Hide some steps in liveness process"
}